packages/kit/src/views/Swap/pages/components/SwapProPriceInfo.tsx

```
@@ -64,7 +68,16 @@ const SwapProPriceInfo = () => {
     };
   }, [priceChange]);
   return (
-    <YStack>
+    <YStack
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)